src/target/tx/devo/t8sg/target_defs.h

```diff
@@ -31,15 +28,20 @@
 #define HAS_AUDIO_UART     1
 #define HAS_MUSIC_CONFIG    1
 
+#define TXTYPE ((char *)0x08003150)
```

Comment:
please add comment on this variable. it will be hard to figure out the meaning via the name without the context of bootloader.